README.rst

```diff
@@ -35,12 +35,12 @@ GitHub repository:
 
     $ git clone https://github.com/cyclus/cymetric
 
-Then build and install:
+Then build and install to the same location Cyclus is installed:
```

Comment:
If we go this approach, should we add a phrase comment that it might not work if it is NOT installed to that same place as Cyclus?